### DISCUSSION TITLE
Welcome to DAFTAR-API-LOKAL-INDONESIA Discussions!

### DISCUSSION THREAD

**Discussion by farizdotid** [farizdotid/DAFTAR-API-LOKAL-INDONESIA#224] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by putra4648** [comment]
Mantap, kenapa baru kebuka sekarang ya? 😂😂

**Comment by rdp77** [comment]
Wow, this is good, can be more active in the discussion here 😊😊

**Comment by vzrenggamani** [comment]
Willing to be maintainer di repo ini bisa kah mas?

**Comment by riodev99** [comment]
Tolong tambah API buatan saya bang tanpa apikey dan limit gratis.

Web : https://forestapi.up.railway.app
Daftar API : https://forestapi.up.railway.app/api

**Comment by 4l4nm4mbu** [comment]
Masih pemula.

**Comment by 666Ajs** [comment]
Masih pemula,bisa ajarin gak!!

**Comment by Kevinmaulanaa** [comment]
Selamat siang mas, bisa di add API Payment Gateway, integrasi pembayaran online. Saat ini sudah support untuk metode CIMB VA, BRI VA dan QRIS.
Berikut dokumentasinya : https://yoo.my.id/api/docs/payments 
Gk perlu verifikasi KTP untuk dapat akses API Payment Gateway ini, karena untuk legal sudah kami persiapkan. Terima kasih

**Reply by sooluh** [reply]
boleh open issue ya mas @Kevinmaulanaa, nanti InsyaAllah saya bantu
